docs/data-sources/eg_official_event_channels.md

```diff
@@ -0,0 +1,60 @@
+---
+subcategory: "EventGrid (EG)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_eg_official_event_channels"
+description: ""
+---
+
+# huaweicloud_eg_official_event_channels
+
+Use this data source to filter EG official event channels within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "channel_name" {}
+
+data "huaweicloud_eg_official_event_channels" "test" {
+  name = var.channel_name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region where the official event channels are located.  
+  If omitted, the provider-level region will be used.
+
+* `name` - (Optional, String) Specifies the channel name used to query specified official event channel.
+
```

Comment:
Why fields `provider_type`, `sort` and `fuzzy_name` are not supported?

Reply:
filter parameter `provider_type` can be supported.